ts/screens/authentication/LandingScreen.tsx

```diff
@@ -333,15 +337,28 @@ export const LandingScreen = () => {
     return (
       <View style={IOStyles.flex}>
         {isSessionExpiredRef.current ? (
-          <LandingCardComponent
-            id={0}
+          <LandingSessionExpiredComponent
             ref={accessibilityFirstFocuseViewRef}
-            pictogramName={"time"}
+            pictogramName={"identityCheck"}
             title={I18n.t("authentication.landing.session_expired.title")}
             content={sessionExpiredCardContent}
-            accessibilityLabel={`${I18n.t(
-              "authentication.landing.session_expired.title"
-            )} ${sessionExpiredCardContent}`}
+            buttonLink={{
+              label: I18n.t(
+                "authentication.landing.session_expired.linkButtonLabel"
+              ),
+              color: "primary",
+              icon: "instruction",
+              onPress: () => {
+                trackHelpCenterCtaTapped(
+                  sessionExpired.toString(),
+                  helpCenterHowToDoWhenSessionIsExpiredUrl,
+                  routeName
+                );
+                openWebUrl("", () => {
```

Comment:
Is it intentional or did you forget to include the link? 

```suggestion
                openWebUrl(helpCenterHowToDoWhenSessionIsExpiredUrl, () => {
```

Reply:
Good catch! It was a try to catch the error message 😅